WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardReadersViewViewModel.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import SwiftUI
+
+struct Manual: Identifiable {
+    let id: Int
+    let image: UIImage
+    let name: String
+    let urlString: String
+}
+
+final class CardReadersViewViewModel {
+    let manuals: [Manual]
+
+    init(manuals: [Manual] = [bbposChipper2XBT, stripeM2, wisepad3]) {
+        self.manuals = manuals
+    }
+}
+
+extension CardReadersViewViewModel {
+    static let bbposChipper2XBT = Manual(
+        id: 0,
+        image: .cardReaderImageBBPOSChipper,
+        name: "BBPOS Chipper 2X BT",
+        urlString: "https://stripe.com/files/docs/terminal/c2xbt_product_sheet.pdf"
+    )
+    static let stripeM2 = Manual(
+        id: 1,
+        image: .cardReaderImageM2,
+        name: "Stripe Reader M2",
+        urlString: "https://stripe.com/files/docs/terminal/m2_product_sheet.pdf"
+    )
+    static let wisepad3 = Manual(
+        id: 2,
+        image: .cardReaderImageWisepad3,
+        name: "Wisepad 3",
+        urlString: "https://stripe.com/files/docs/terminal/wp3_product_sheet.pdf"
+    )
```

Comment:
Perhaps all this could relate to/be defined on [the `CardReaderType` enum](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Hardware/Hardware/CardReader/CardReaderType.swift)?

Reply:
Addressed on https://github.com/woocommerce/woocommerce-ios/pull/7227